### PR TITLE
Add Customer Authentication OTP

### PR DESCRIPTION
**Pull request by justnero-ru**

### CODE REVIEW

File: spec/components/schemas/Customer.yaml [hunk 1]
```diff
@@ -61,4 +61,4 @@ properties:
       - $ref: "#/components/schemas/NotesLink"
       - $ref: "#/components/schemas/DefaultPaymentInstrumentLink"
       - $ref: "#/components/schemas/LeadSourceLink"
-
+      - $ref: "#/components/schemas/WebsiteLink"
```
Comment (slavcodev) [inline spec/components/schemas/Customer.yaml]: How website is related to customer? I don't see somethign like `websiteId` in resource.

Comment (slavcodev) [reply spec/components/schemas/Customer.yaml]: Customer can belong to many websites AFAIK

Comment (justnero-ru) [reply spec/components/schemas/Customer.yaml]: Please see #703 Branch is rebased

Comment (slavcodev) [reply spec/components/schemas/Customer.yaml]: Do you check DB if we can assign website to all customers?

Comment (justnero-ru) [reply spec/components/schemas/Customer.yaml]: Sure, actually website feature is live

**Pull request merged**